### PR TITLE
Bunch of enhancements

### LINKED ISSUE #21: Categories: Custom Button Text
Each category should be allowed to determine custom buy button text as category X may not be a book for example.

### LINKED ISSUE #22: Allow both links to external books and buttons to download files
Some of our books have online editions in Pressbooks, but also PDF versions for download.

Currently, if remote link is entered, format downloads are not presented. Would like to have both display, if available.

### LINKED ISSUE #23: Copyright/Licensing Information
Allow entering and display of copyright/licensing information for books/chapters.

### LINKED ISSUE #26: Hide downloads if no files to download
Some of our books are remote. Would like to hide the download count for books without files to download.

### PR DESCRIPTION
**Pull request by ajrbyers**
Closes #21
Closes #22
Closes #23 
Closes #26

### CODE REVIEW

File: templates/books/book_detail.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
             {% if not books %}<p><br /><a class="button" href="{% url 'books_index' %}"><i class="fa fa-arrow-left">&nbsp;</i>Back to Listing</a></p>{% endif %}
         </div>
-        <div class="{% if books %}large-9 columns{% else %}large-7 columns{% endif %}">
+        <div class="{% if books %}large-9 columns{% else %}large-7 columns{% endif %}{% if not books and not book.format_set.all %} end{% endif %}">
```
Comment (mauromsl) [inline templates/books/book_detail.html:10]: .exists would be better than .all here

File: models.py [hunk 2]
```diff
@@ -354,6 +364,12 @@ class Chapter(models.Model):
     filename = models.CharField(
         max_length=255,
     )
+    license_information = models.TextField(
+        blank=True,
+        null=True,
+        help_text="Add copyright and/or license information here. "
+                  "If left blank will display book version of this field.",
+    )
```
Comment (mauromsl) [inline models.py:372]: Is there a reason to not use submission.Licence here?

**Review by mauromsl — CHANGES_REQUESTED** [review]
One minor nitpick and one question

File: models.py (hunk 2 above)
Comment (ajrbyers) [reply models.py:372]: Flexibility - book licenses aren't the same as articles.

**Pull request merged**